pages/index.tsx

```diff
@@ -1,26 +1,18 @@
-import Demo from '@/components/views/Demo/Demo';
-import { Inter } from 'next/font/google';
-import React from 'react';
+import Icon from "@/components/shared/Icon/Icon";
+import Demo from "@/components/views/Demo/Demo";
+import { Inter } from "next/font/google";
 
-const inter = Inter({ subsets: ['latin'] });
+const inter = Inter({ subsets: ["latin"] });
```

Comment:
We've already added fonts in `_app.tsx`. This line is never used
```suggestion
const inter = Inter({ subsets: ["latin"] });
```

Reply:
removed const inter